lib/core/juliaManager.cpp

```diff
@@ -168,4 +170,21 @@ std::any juliaCallAny(const std::function<std::any()>& fun) {
     return res;
 }
 
+void juliaHandleExceptions() {
+    jl_value_t* exc = jl_exception_occurred();
+    if (exc) {
+        ERROR_F("Caught Julia exception");
+
+        // Call showerror(io, exc)
+        jl_function_t* showerror = jl_get_function(jl_base_module, "showerror");
+        jl_call2(showerror, jl_stderr_obj(), exc);
+        // jl_flush_cstdio();
+        jl_print_backtrace();
+
+        std::fflush(stdout);
+        std::fflush(stderr);
+        std::abort();
```

Comment:
I'm not familiar with possible julia exceptions -- do we *always* want to abort?
Consider throwing here to give other stages/threads a chance to clean up.